.clippy_allowed_lints

```diff
@@ -0,0 +1,39 @@
+# This is a list of allowed Clippy rules for the purposes of gradual migration.
+# Unfortunately, Clippy doesn't support allowing lints in a configuration file, so we roll our own.
+# See https://doc.rust-lang.org/stable/clippy/configuration.html
+
+borrow_interior_mutable_const
```

Comment:
I wonder what is the difference between this custom config file, vs using `clippy.toml` and setting these rules to `false`? The latter means that we can pick it up from RA runs as well.

Reply:
I would love to use the single config for these for the benefit you mentioned, however `clippy.toml` only allows to set configuration values for specific lints, not allowing/warning/denying any list. Whenever you try to specify a lint that's only allow/deny in the `clippy.toml`, the Clippy errors out with an 'unknown lint' error.